### PR TITLE
42 tour organize your visit

### LINKED ISSUE #42: Tour - implement organize your visit columns
Create styles for the columns in the home page of the tour.
We won't create a new block for this columns, but reuse the OOTB columns block with an [option] (https://www.hlx.live/developer/markup-sections-blocks#block-options) 

These type of columns support:
1. Icon
2. Text

You can see the existing look and feel in this url: https://www.realmadrid.com/en/tour-bernabeu 

Screenshot 2023-05-05 at 16 59 21

Screenshot 2023-05-05 at 16 59 31

### PR DESCRIPTION
**Pull request by etomaello**
the column block customization for organize-your-visit section has been developed with mobile first approach, it's a two column table **Icon** | **_Text_**

fragment file location: https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/Shared%20Documents/sites/realmadrid/tour/es/fragments/organize-your-visit.docx?d=wd88d04a373714103923ddd8322ce21ad&csf=1&web=1&e=tgTnCU

Fix #42 

Test URLs:
- Before: https://main--realmadrid--hlxsites.hlx.page/tour
- After: https://42-tour-organize-your-visit--realmadrid--hlxsites.hlx.page/tour

### CODE REVIEW

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import url("organize-visit.css");
```
Comment (chicharr) [inline blocks/columns/columns.css:1]: I'm not sure about using a separate .css file for this. Sometimes it takes more time to open a second connection to the server to download a second file, than downloading a single file with 100 lines more.
I think we better keep it in the same file.
FYI, there is also a `loadCSS` function in lib-franklin.js which allows you to load CSS dynamically from javascript, i.e. if the `decorateOrganizeVisit`

**Review by chicharr — APPROVED** [review]
It looks great @etomaello. Just few small comments

File: blocks/columns/organize-visit.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
+.columns.organize-visit svg {
+    display: none;
+}
+
+.columns.organize-visit a {
+    text-decoration: underline;
```
[the remark this answers is not in the document]
Comment (etomaello) [reply blocks/columns/organize-visit.css:6]: @chicharr no, it's underlined on mouseover

File: blocks/columns/columns.css (hunk 1 above)
Comment (etomaello) [reply blocks/columns/columns.css]: I've removed the import and used loadcss instead, also tried to load the component twice and css was loaded only once.

**Pull request merged**